src/rustdoc-internals/htmldocck.md

```diff
@@ -0,0 +1,101 @@
+# `rustdoc` tests
```

Comment:
It's slightly unusual to have section heading != file name.

Reply:
I did this to avoid using a filepath we might want to use later for a general overview of all `rustdoc*` test suites.

while moving files can be done smoothly with redirects, swapping them will always lead to confusing.